DataAccess/Models/Release.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DataAccess.Models
+{
+    public class Release
+    {
+        public int ReleaseId { get; set; }
+        public string ReleaseName { get; set; }
+        public DateTime StartDate { get; set; }
+        public DateTime EndDate { get; set; }
+        public virtual ICollection<Sprint> Sprints { get; set; }
+        public int SprintStatusId { get; set; }
+        public SprintStatus SprintStatus { get; set; }

```

Comment:
Why does release have SprintStatus?

Reply:
if we need to maintain status then we should re-name it to Status and use in both release and sprint